scripts/lib/install_cassandra.sh

```diff
@@ -28,10 +28,22 @@ modify_server_hostname() {
   sudo sed -i 's/<public name>/127.0.0.1/' /etc/cassandra/cassandra-env.sh
 }
 
+
+install_python_driver() {
+  command -v pip3 > /dev/null 2>&1;
+  if [[ $? -eq 1 ]]; then
+    echo "Installing pip..."
+    sudo apt-get -y install python3-pip
+  fi
+  echo "Installing cassandra-driver..."
+  pip install cassandra-driver
+}
```

Comment:
If you're using pip to install Cassandra, I prefer that you install this in Python's own virtual environment.

Reply:
True